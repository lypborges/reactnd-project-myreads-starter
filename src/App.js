import React, { Component } from "react";
import BookSearch from "./ui/routes/BookSearch";
import BookList from "./ui/routes/BookList";
import { BrowserRouter as Router, Route } from "react-router-dom";

import "./App.css";

class BooksApp extends Component {
  render() {
    return (
      <Router>
        <div>
          <Route exact path="/" component={BookList} />
          <Route exact path="/search" component={BookSearch} />
        </div>
      </Router>
    );
  }
}

export default BooksApp;
