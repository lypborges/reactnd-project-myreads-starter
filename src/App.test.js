import App from "./App";

describe("App", () => {
  test("App setup books on shelf", () => {
    expect(true);
  });
});
