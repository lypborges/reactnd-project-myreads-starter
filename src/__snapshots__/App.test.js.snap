// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render without books 1`] = `
<div>
  <div
    className="ant-layout"
  >
    <div
      className="main-header ant-layout-header"
    >
      <i
        className="anticon anticon-book"
        style={
          Object {
            "color": "white",
            "fontSize": 50,
          }
        }
      />
      <h1
        className="main-title"
      >
        My reads
      </h1>
    </div>
    <div
      className="main-content ant-layout-content"
    >
      <div
        className="list-spinner"
      >
        <div
          className="ant-spin ant-spin-lg ant-spin-spinning"
        >
          <span
            className="ant-spin-dot ant-spin-dot-spin"
          >
            <i />
            <i />
            <i />
            <i />
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
